src/js/user-profile/components/FormItem.jsx

```diff
@@ -0,0 +1,58 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import moment from 'moment';
+
+import { dateDiffDesc } from '../../common/utils/helpers';
+import { formTitles, formLinks } from '../helpers';
+
+class FormItem extends React.Component {
+  render() {
+    const savedFormData = this.props.savedFormData;
+    const formId = savedFormData.form;
+    const { last_updated: lastSavedTime, expires_at: expirationTime } = savedFormData.metadata;
+    const lastSavedDate = moment.unix(lastSavedTime);
+    const expirationDate = moment.unix(expirationTime);
+    const isExpired = moment(expirationDate).isBefore(moment().startOf('minute'));
+    const activeView = (
+      <div className="card information">
+        <div className="row small-collapse saved-form-information">
+          <div className="small-12 large-8 columns">
+            <h5 className="saved-form-title">Application for {formTitles[formId]}</h5>
+            {!!lastSavedDate && !!expirationDate && <div>
+              <span className="saved-form-item-metadata">Last saved on {moment(lastSavedDate).format('MMMM D, YYYY')}</span>
+              <span className="saved-form-item-metadata saved-form-item-expires">Expires in {dateDiffDesc(expirationDate)}</span>
+            </div>}
+          </div>
+          <div className="small-12 large-4 columns">
+            <a className="usa-button-primary resume-saved-application" href={formLinks[formId]}>Resume application</a>
```

Comment:
This avoids the broken link issue, since we just put them on the intro page, but it would be a better experience to get them on the page they left off on without the extra click.

Maybe we should make a separate issue for that?

Reply:
Good idea, [created](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/4537).